java/ql/src/Telemetry/AutomodelExtractCandidates.ql

```diff
@@ -0,0 +1,39 @@
+/**
+ * Surfaces the endpoints that are not already known to be sinks, and are therefore used as candidates for
+ * classification with an ML model.
+ *
+ * Note: This query does not actually classify the endpoints using the model.
+ *
+ * @name Automodel candidates
+ * @description A query to extract automodel candidates.
+ * @kind problem
+ * @severity info
+ * @id java/ml-powered/extract-automodel-candidates
```

Comment:
Suggestion: just use `ml` rather than `ml-powered` so that we don't confuse these results in our internal data with those from the legacy ATM queries.

Reply:
Done :)